assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+
+{
+  "id": "campersite",
+  "title": {
+    "en": "Campersites"
+  },
+  "shortDescription": {
+    "en": "Find sites to spend the night with your camper"
+  },
+  "description": {
+    "en": "This site collects all official camper stopover places and places where you can dump grey and black water. You can add details about the services provided and the cost. Add pictures and reviews. This is a website and a webapp. The data is stored in OpenStreetMap, so it will be free forever and can be re-used by any app."
+  },
+  "language": [
+    "en"
+  ],
+  "maintainer": "",
+  "icon": "./assets/themes/campersites/caravan.svg",
+  "version": "0",
+  "startLat": 43.14,
+  "startLon": 3.14,
+  "startZoom": 14,
+  "widenFactor": 0.05,
+  "socialImage": "./assets/themes/campersites/Bar%C3%9Fel_Wohnmobilstellplatz.jpg",
+  "layers": [
+    {
+      "id": "caravansites",
+      "name": {
+        "en": "Camper sites"
+      },
+      "minzoom": 12,
+      "overpassTags": {
+        "and": [
+          "tourism=caravan_site"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Camper site {name}"
+        },
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "Free camper site {name}"
+            }
+          }
+        ]
+      },
+      "description": {
+        "en": "camper sites"
+      },
+      "tagRenderings": [
+        "images",
+        {
+          "render": {
+            "en": "power supply"
+          },
+          "question": {
+            "en": "Does this place have a power supply?"
+          },
+          "freeform": {
+            "key": "power_supply",
+            "addExtraTags": [
+              "power_supply=yes"
```

Comment:
This is a bit weird. The freeform text field has a value "power_supply", but you will overwrite the value by the user by using `power_supply=yes`. If you want to leave the freeform field, delete this extraTag, or use a different key e.g. "fixme=power_supply was filled in freeformly by a mapcomplete user, must be checked"

Reply:
I think I only wanted to show the "special values" here if they were filled in, not overrule them. I removed the addExtraTags